frontend/src/components/pages/WaitlistExperience/index.tsx

```diff
@@ -47,10 +50,39 @@ const WaitlistExperiencePage = ({
       </Center>
     );
 
-  const handleStepNavigation = (stepsToMove: number) => {
+  const registerWaitListCampers = async (): Promise<boolean> => {
+    setRegistrationLoading(true);
+    try {
+      await CamperAPIClient.waitlistCampers(
+        campers,
+        Array.from(selectedCampSessions, (campSession) => campSession.id),
+      );
+      setRegistrationLoading(false);
+      return true;
+    } catch (error: Error | unknown) {
+      setRegistrationLoading(false);
+      let errorMessage =
+        "Unable to create a checkout session. Please try again.";
+      if (error instanceof Error) {
+        errorMessage = error.message;
```

Comment:
My only issue with this is that some of the mailer's error messages might be cryptic and mean nothing to a registrant. I'd say delete this if statement and the errorMessage var (lines 64-68). In the toast's description, write "Unable to create a checkout session. Please review information try again." (similar to what you had before but with the review information prompt).